__tests__/push/index.test.ts

```diff
@@ -262,7 +262,12 @@ heartbeat.monitors:
           await new Promise(r => setTimeout(r, 20));
           req.on('data', chunks => {
             const schema = JSON.parse(chunks.toString()) as APISchema;
-            res.write(JSON.stringify(schema.monitors[0].name) + '\n');
+            res.write(
+              JSON.stringify(
+                schema.monitors.length + ' monitors created successfully'
+              ) + '\n'
+            );
+            res.write(JSON.stringify('1 monitor updated successfully') + '\n');
```

Comment:
better to write something specific to monitor related details other than static string.